package/.shellcheckrc

```diff
@@ -0,0 +1,15 @@
+# vim: set ft=sh:
+# Configuration for Shellcheck in package recipes
+# See <https://github.com/koalaman/shellcheck/blob/master/shellcheck.1.md#rc-files>
+
+# Do not warn about unguarded `cd`s since recipes are always executed
+# with the -e flag set
+disable=SC2164
+
+# Do not warn about unused variables since metadata fields are expected to
+# not necessarily be used in the recipes
+disable=SC2034
```

Comment:
Unfortunately there is no direct way of doing that yet, as far as I know. Issue koalaman/shellcheck#2020 proposes to add an `is_used` directive that would do exactly that. Two workarounds come to mind:

1. `export` all the metadata fields in recipes. IMO, this would add too much noise.
2. Instead of passing each recipe as-is to shellcheck, append dummy lines that use the fields in question, and pass the result to shellcheck. To do that we’ll need to write a separate script that applies this workaround to each file as needed and then calls shellcheck.